modules/light-clients/08-wasm/types/update_test.go

```diff
@@ -78,7 +77,9 @@ func (suite *TypesTestSuite) TestUpdateState() {
 					err := json.Unmarshal(sudoMsg, &msg)
 					suite.Require().NoError(err)
 
-					store.Set(host.ClientStateKey(), msg.UpdateState.ClientMessage.Data)
+					expectedClientStateBz = wasmtesting.CreateMockClientStateBz(suite.chainA.Codec, suite.codeHash)
+					store.Set(host.ClientStateKey(), expectedClientStateBz)
+
```

Comment:
I think in this test case I would update the client state with the data that is received in the message, so that it actually does what the test case says. So something like this:

```golang
bz := store.Get(host.ClientStateKey())
suite.Require().NotEmpty(bz)
clientState := clienttypes.MustUnmarshalClientState(suite.chainA.Codec, bz).(*types.ClientState)
clientState.Data = msg.UpdateState.ClientMessage.Data
store.Set(host.ClientStateKey(), clienttypes.MustMarshalClientState(suite.chainA.App.AppCodec(), clientState))
```